state-sync/inter-component/event-notifications/src/lib.rs

```diff
@@ -146,6 +149,15 @@ impl EventSubscriptionService {
                 .or_insert_with(|| HashSet::from_iter([subscription_id].iter().cloned()));
         }
 
+        for event_tag in event_v2_tags {
```

Comment:
nit: for consistency, I'd put a single line comment above this block (so that it matches the others), e.g.:
```
 // Update the event v2 tag subscriptions to include the new subscription
```